crates/piwis-val/src/lib.rs

```diff
@@ -88,6 +88,8 @@ pub struct VehicleData {
     pub operating_time: UnitString,
     #[serde(rename = "ORDERTYPE")]
     pub order_type: String,
+    #[serde(default, rename = "MODEL")]
+    pub model: String,
```

Comment:
default because its sometimes undefined? in that case rather than a String you can use an [Option](https://doc.rust-lang.org/std/option/enum.Option.html). And since everything in this is already applying the SCREAMING_SNAKE_CASE rule, you don't need to rename it to MODEL. the other fields that have the rename defined in here are done so because they have the _ and it needs to be removed to match the xml.

```suggestion
    pub model: Option<String>,
```

Reply:
Yes that works, I think I overcomplicated it.